crates/consensus/consensus/src/lib.rs

```diff
@@ -20,7 +20,6 @@ use std::fmt::Debug;
 
 #[cfg(not(feature = "std"))]
```

Comment:
Are these feature gates still necessary since the std feature array is empty?

Reply:
https://github.com/paradigmxyz/reth/pull/9663#discussion_r1703946604